frontend/src/routes/Infrastructure/Automations/AnsibleAutomationsForm.tsx

```diff
@@ -89,6 +90,7 @@ export function AnsibleAutomationsForm(props: {
     isViewing: boolean
 }) {
     const { t } = useTranslation(['cluster', 'common', 'credentials', 'create'])
+    const { t: tCredential } = useTranslation(['credentials'])
```

Comment:
I guess I could reorder the files in the top useTranslation, so that 'credentials' is first, but then I would need to edit every string in the file using the default 'cluster' and add a 'cluster:' at the front of the string. 

...the translation happening at validation.ts assumes that t('credentials') is being passed in.

Reply:
I think you can just set the string keys to include the `credentials:` reference